libs/patches/ua-parser-js+1.0.2.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/node_modules/ua-parser-js/package.json b/node_modules/ua-parser-js/package.json
```

Comment:
this patch is here as Rollup `preserveModules` adds a commonjs runtime for modules that has dynamic exports (like `ua-parser-js` which does `modules.exports = ` in IIFE). This could work, but rollup puts its commonjs runtime together with a module stub in `_virtaul` folder which breaks Chromium as folder names starting with `_` are prohibited in that engine.

Reply:
Ugly thing, but I get it. As we talked about, we should remove that package from dependencies in separate PR.